### DISCUSSION TITLE
Watches with xDrip+

### DISCUSSION THREAD

**Discussion by Joachim-dot** [Navid200/xDrip#118] [Watches]
Hello, I need your help: Which smartwatches can I now pair with xDrip +?  xDrip + is installed on an Android 7.0 smartphone. Is there a list for this?

**Comment by Jorge-1234** [comment]
Miband

**Comment by tripmaker-j** [comment]
Hi, I use mine for display on a Fitbit Versa Lite using the Glance app.

**Comment by Jorge-1234** [comment]


**Reply by Navid200** [reply]
These forums are retired.
Please use these instead:
https://github.com/NightscoutFoundation/xDrip/discussions

I will delete your post now.
